blueocean-core-js/src/js/ToastService.js

```diff
@@ -43,14 +51,31 @@ export class ToastService {
      * @param toast
      */
     removeToast(toast) {
-        this.toasts = this.toasts.filter((item) => {
-            return toast.id !== item.id;
-        });
+        this.toasts = this.toasts.filter((item) =>
```

Comment:
honestly not sure which of these i prefer... seems to be easier to debug the bracket-enclosed version, would definitely prefer this changed style if it was easier to debug